plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/reflection/HtmlReflection.java

```diff
@@ -465,4 +447,12 @@ private static String getFieldTitleInBlock(Field field) {
         }
         return "";
     }
+
+    public <T> T getElementByField(Object parentObject, Field field) throws ElementDescriptionException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1161))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=101&issues=AWbDgFYRLpqqSUlctfyH&open=AWbDgFYRLpqqSUlctfyH)

[ISSUE_KEY]: AWbDgFYRLpqqSUlctfyH